src/unifrac_task_noclass.cpp

```diff
@@ -2210,8 +2131,510 @@ void SUCMP_NM::UnifracVawUnnormalizedUnweightedTask<TFloat>::_run(unsigned int f
 
       }
     }
+}
+
+} // end namespace
+
+/*
+ * Create concrete implementation wrappers
+ */
+
+template<>
```

Comment:
I haven't encountered this before, no type needed?

Reply:
...is it because the called function is meta?

Reply:
This is how you force the generation of a specific type when you do separate compilation.